src/DataGrid.tsx

```diff
@@ -129,8 +145,6 @@ export interface DataGridProps<R, K extends keyof R, SR = unknown> {
    */
   /** Toggles whether filters row is displayed or not */
   enableFilters?: boolean;
-  /** Toggles whether cells should be autofocused */
-  enableCellAutoFocus?: boolean;
```

Comment:
What was this for at first? Making (0,0) cell selected upon mounting the grid?

Reply:
Correct. I think we should not steal the focus and let the user select the first cell using `grid.selectCell`